chart/templates/extra-objects.yaml

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
Isn't this `---`   extra ?

Reply:
That is due to the linting requirements on our repo's can remove here though.
```suggestion
```